pkg/config/config.go

```diff
@@ -19,8 +19,10 @@ type Config struct {
 }
 
 type Garden struct {
-	Name       string `yaml:"name"`
-	Kubeconfig string `yaml:"kubeconfig"`
+	Name            string `yaml:"name"`
+	Kubeconfig      string `yaml:"kubeconfig"`
+	ShootDomain     string `yaml:"shootDomain"`
+	DashboardDomain string `yaml:"dashboardDomain"`
```

Comment:
Why don't we add a generic Patter Matching mechanism which also can handle technicalID. Add an Array of e.g `shootRegexps` or `shootMatchPatterns`. The implementation should be generic and could look like https://play.golang.org/p/q7JwrHDu283.

Reply:
Yes, this sounds like a good Idea. We should document this well, so that users know how to configure their gardens. Would you add this per garden or global? If we have separate regexps per garden, we could keep the functionality of also targeting across multiple gardens with shoot / dashboard urls. Maybe we can have both?